### PR TITLE
Mattbranch

### PR DESCRIPTION
**Pull request by Matt-Oliva**

### CODE REVIEW

File: src/components/App/App.jsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ function App() {
         <div className="selling-points">
           <div className="point">
 
-            <h1>Service 1</h1>
+            <h1>Testing</h1>
```
Comment (rickyrojas1) [inline src/components/App/App.jsx]: Can you change this back to `Service 1`

Comment (Matt-Oliva) [reply src/components/App/App.jsx]: i got you booboo

**Comment by rickyrojas1** [comment]
Can you make some creative services names

File: src/components/App/App.jsx (hunk 1 above)
Comment (Matt-Oliva) [reply src/components/App/App.jsx]: How are these?

**Pull request merged**